DentOS_Framework/DentOsTestbed/src/dent_os_testbed/constants.py

```diff
@@ -68,12 +68,13 @@
     "suite_acl_performance": "ACL Performance related tests",
     "suite_poe_cli": "POE related tests with the cli tool",
     "suite_cleanup": "Test marker for test cleanup",
+    "suite_ipv4": "IPv4 related tests",
 }
 
 PYTEST_SUITE_GROUPS = {
     "suite_group_test": ["suite_test", "suite_feature_f1", "suite_feature_f2"],
     "suite_group_clean_config": ["suite_clean_config"],
-    "suite_group_l3_tests": ["suite_iproute2", "suite_arp"],
+    "suite_group_l3_tests": ["suite_iproute2", "suite_arp", "suite_ipv4"],
```

Comment:
those tests might require a different setup and it won't be possible to run them in the same group.